src/PIL/PngImagePlugin.py

```diff
@@ -892,6 +905,7 @@ def append(fp, cid, *data):
 Image.register_open(PngImageFile.format, PngImageFile, _accept)
 Image.register_save(PngImageFile.format, _save)
 
-Image.register_extension(PngImageFile.format, ".png")
+Image.register_extensions(PngImageFile.format, [".png", ".apng"])
 
 Image.register_mime(PngImageFile.format, "image/png")
+Image.register_mime(PngImageFile.format, "image/apng")
```

Comment:
This overwrites the `register_mime()` call on the previous line. Maybe `PngImageFile.format` should be changed to `'APNG'`?

Reply:
Resolved by #3525